### PR TITLE
#101 Dont allow imports method for test_package

### LINKED ISSUE #101: [conan center] check there is no imports in test_package/conanfile.py
seems like `imports` method is not needed in **test_package/conanfile.py**, if we use `run_environment`?

### PR DESCRIPTION
**Pull request by uilianries**
closes #101

Wiki:

#### **<a name="KB-H034">#KB-H034</a>: "TEST PACKAGE - IMPORTS"**

The method [imports](https://docs.conan.io/en/latest/reference/conanfile/methods.html#imports) helps the test package stage copying all dynamic libraries and special resources. However, the [run_environment](https://docs.conan.io/en/latest/reference/conanfile/other.html#running-commands) parameter, used when running commands, is able to solve all paths to the dynamic libraries installed in your cache.

### CODE REVIEW

**Comment by Croydon** [comment]
I'm thinking the name is confusing. When I read it I was thinking that Python imports wouldn't be allowed (which is impossible as Conan needs to be imported)

**Comment by uilianries** [comment]
> I'm thinking the name is confusing. When I read it I was thinking that Python imports wouldn't be allowed (which is impossible as Conan needs to be imported)

**Comment by Croydon** [comment]
What about `No imports()`?

**Comment by uilianries** [comment]
> What about `No imports()`?

sounds better!

**Comment by uilianries** [comment]
All conflicts has been solved.

File: hooks/conan-center.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
              "KB-H029": "TEST PACKAGE - RUN ENVIRONMENT",
              "KB-H030": "CONANDATA.YML FORMAT",
              "KB-H031": "CONANDATA.YML REDUCE",
+             "KB-H034": "NO IMPORTS()",
```
Comment (danimtb) [inline hooks/conan-center.py]:
```suggestion
             "KB-H034": "TEST PACKAGE - NO IMPORTS()",
```

Comment (danimtb) [reply hooks/conan-center.py]: I am committing this @uilianries. Thanks!

Comment (danimtb) [reply hooks/conan-center.py]: ok, seems I don't have permission over your branch

File: tests/test_hooks/conan-center/test_conan-center.py [hunk 2]
```diff
@@ -218,6 +218,7 @@ def test(self):
         self.assertIn("[TEST PACKAGE FOLDER (KB-H024)] OK", output)
         self.assertIn("[TEST PACKAGE - RUN ENVIRONMENT (KB-H029)] OK", output)
         self.assertIn("[EXPORT LICENSE (KB-H023)] OK", output)
+        self.assertIn("[NO IMPORTS() (KB-H034)] OK", output)
```
Comment (danimtb) [inline tests/test_hooks/conan-center/test_conan-center.py]:
```suggestion
        self.assertIn("[TEST PACKAGE - NO IMPORTS() (KB-H034)] OK", output)
```

File: tests/test_hooks/conan-center/test_conan-center.py [hunk 3]
```diff
@@ -444,6 +445,29 @@ def test(self):
...
+            def test(self):
+                self.run("echo bar", run_environment=True)
+        """)
+
+        tools.save('test_package/conanfile.py', content=conanfile_tp)
+        tools.save('conanfile.py', content=self.conanfile)
+        output = self.conan(['create', '.', 'name/version@user/test'])
+        self.assertIn("[TEST PACKAGE FOLDER (KB-H024)] OK", output)
+        self.assertIn("[TEST PACKAGE - RUN ENVIRONMENT (KB-H029)] OK", output)
+        self.assertIn("ERROR: [NO IMPORTS() (KB-H034)] The method `imports` is not " \
```
Comment (danimtb) [inline tests/test_hooks/conan-center/test_conan-center.py]:
```suggestion
        self.assertIn("ERROR: [TEST PACKAGE - NO IMPORTS() (KB-H034)] The method `imports` is not " \
```

**Comment by uilianries** [comment]
Done! :)

**Pull request merged**